guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -88,12 +88,12 @@ For smaller content setups, or if you can afford longer downtimes, you can proce
 
 For {SmartProxies}, you can choose to deploy new 6.10 {SmartProxies} rather than to upgrade.
 
-For future upgrades following {Project} 7.0, you will be required to upgrade the operating system from RHEL 7 to RHEL 8 on your {ProjectServer}s and {SmartProxies}.
+For future upgrades following {Project} 6.11, you will be required to upgrade the operating system from RHEL 7 to RHEL 8 on your {ProjectServer}s and {SmartProxies}.
```

Comment:
Satellite 6.11 will still run on RHEL 7 but you must upgrade to RHEL 8 while on 6.11 before you can upgrade to 6.12. I'm not sure `following` is clear enough in conveying this message.